inc/formanswer.class.php

```diff
@@ -1551,4 +1560,33 @@ public static function getMyLastAnswersAsValidator($limit = 5) {
 
       return $DB->request($request);
    }
+
+   /**
+    * Find if the HTTP request will display the display preferences edition form
+    * The ID of the form is in $_SESSION['formcreator']['form_search_answers']
+    * @see PluginFormcreatorFormAnswer::showForForm()
+    *
+    * @return boolean true if search options shall show fields for a specific form
+    */
+   private function showFieldSearchOptions() {
+      global $CFG_GLPI;
+
+      if (!isset($_SESSION['formcreator']['form_search_answers'])) {
+         return false;
+      }
+
+      if (strpos($_SERVER['REQUEST_URI'], $CFG_GLPI['root_doc'] . '/ajax/common.tabs.php') !== 0) {
+         return false;
+      }
+
+      if (isset($_GET['displaytype']) && $_GET['displaytype'] == PluginFormcreatorFormAnswer::class) {
+         // return true;
```

Comment:
enabling this gives a way to display or not the fields specific to the form. Not sure this is needed (may break if a form has 10 fields and display prefs are configured to show all of them, but an other has only 5)